app/models/game.rb

```diff
@@ -1,6 +1,7 @@
 class Game < ApplicationRecord
 
   scope :available, -> { where(black_player_id: nil) }
+  scope :current, -> { where(black_player_id: !nil) }
```

Comment:
@JefeNelson I created a game and didn't see my game under current game. Did it work for you?

@xelzi we may need to also revisit the available game scope and add conditions in the index view so that only a user who didn't create a game and has the option to join should see games available